zio-schema/shared/src/main/scala/zio/schema/validation/Time.scala

```diff
@@ -0,0 +1,139 @@
+package zio.schema.validation
+
+import scala.annotation.tailrec
+import scala.collection.mutable
+
+trait Time {
+
+  /**
+   *
+   * Format is the same is the one used by the java.time.format.DateTimeFormatter class.
+   * This is copied from there.

```

Comment:
Unfortunately we can't copy even comments from the JDK without running into issues. I'd suggest linking to the Javadoc or re-creating a similar table that documents the format accepted.

Reply:
Changed it.